### PR TITLE
🔁 Clarify about contact import columns

### PR DESCRIPTION
**Pull request by norkans7**

### CODE REVIEW

**Comment by rowanseymour** [comment]
@norkans7 can you add a screen shot por favor

**Comment by norkans7** [comment]
screenshot-1
screenshot-2

**Comment by norkans7** [comment]
screenshot-1
screenshot-2

File: templates/contacts/contact_import.haml [hunk 1]
```diff
@@ -32,6 +32,86 @@
               -trans "+250788111222"
             %td
               -trans "Jane Doe"
+        -if show_form
```
Comment (norkans7) [inline templates/contacts/contact_import.haml]: these tables are hidden when selecting the fields to include

File: templates/contacts/contact_import.haml [hunk 2]
```diff
@@ -14,12 +14,12 @@
           -trans "You can import your contacts from an XLS file you create in Excel."
 
         %p
-          -trans "Each row should have the phone number (including country code) and the name of the contact if available."
+          -trans "Each row should have the phone number as URN:tel (including country code) and the name of the contact if available."
```
Comment (rowanseymour) [inline templates/contacts/contact_import.haml]: maybe use italics for emphasis `<i>URN:tel</i>` ?

File: templates/contacts/contact_import.haml [hunk 3]
```diff
@@ -32,6 +32,86 @@
               -trans "+250788111222"
             %td
               -trans "Jane Doe"
+        -if show_form
+
+          %p
+            -trans "You can also import custom contact fields values in XLS file for columns with prefix 'Field:'."
```
Comment (rowanseymour) [inline templates/contacts/contact_import.haml]: `...prefix <i>Field:</i>, for example:`

File: templates/contacts/contact_import.haml [hunk 4]
```diff
@@ -32,6 +32,86 @@
...
+              %th
+                -trans "Imported as"
+              %th
+                -trans "Details"
+
+            %tr
+              %td
+                -trans "Name"
+              %td
+                -trans "contact name"
```
Comment (rowanseymour) [inline templates/contacts/contact_import.haml]: **C**ontact name

File: templates/contacts/contact_import.haml [hunk 5]
```diff
@@ -32,6 +32,86 @@
...
+              %td
+
+            %tr
+              %td
+                -trans "Language"
+              %td
+                -trans "contact language"
```
Comment (rowanseymour) [inline templates/contacts/contact_import.haml]: as above

File: templates/contacts/contact_import.haml [hunk 6]
```diff
@@ -32,6 +32,86 @@
...
+              %td
+                -trans "Only valid ISO639-3 code supportted"
```
Comment (rowanseymour) [inline templates/contacts/contact_import.haml]: `..codes supported`

File: templates/contacts/contact_import.haml [hunk 7]
```diff
@@ -32,6 +32,86 @@
...
+                  You will have the option to choose the columns to import and to which field to import them.
+                  custom_field is anything containing only letters, numbers, hyphens or spaces.
+
+            %tr
+              %td
+                Group:Group Name
+              %td
+                -trans "IGNORED"
+              %td
+                Columns starting by Group: prefix are all ignored by the %import
```
Comment (rowanseymour) [inline templates/contacts/contact_import.haml]: `Columns starting with the prefix <i>Group:</i> are ignored during the import`

**Comment by norkans7** [comment]
screenshot-3

**Pull request merged**